webview/src/test/nodes.ts

```diff
@@ -0,0 +1,3 @@
+export const idToNodeNode = (id: string) =>
+  // eslint-disable-next-line unicorn/prefer-query-selector
```

Comment:
This is needed to pass Windows tests. When not using `getElementById` we need a `CSS.escape` because of all the funky ids we have in both the experiments table and plots. `CSS.escape` does not seem to create a valid id query on Windows unfortunately.